src/applications/personalization/dashboard-2/components/health-care/Prescriptions.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import HealthCareCard from './HealthCareCard';
+import { recordDashboardClick } from 'applications/personalization/dashboard/helpers';
+
+import { mhvUrl } from '~/platform/site-wide/mhv/utilities';
+
+export const Prescriptions = ({ authenticatedWithSSOe, prescriptions }) => {
+  let filteredPrescriptions;
+  let latestPrescription;
+  let nrRefillsInProgress;
+
+  if (prescriptions?.length) {
+    const thirtyDaysAgo = moment()
+      .endOf('day')
+      .subtract(30, 'days');
+    const statuses = new Set(['refillinprocess', 'submitted']);
+
+    filteredPrescriptions = prescriptions
+      .filter(p => statuses.has(p.attributes?.refillStatus))
+      .filter(
+        p =>
+          moment(p.attributes?.refillSubmitDate).isAfter(thirtyDaysAgo) ||
+          moment(p.attributes?.refillDate).isAfter(thirtyDaysAgo),
+      );
+
+    // GET LATEST DATE TO MAKE SURE LATEST PRESCRIPTION IS CORRECT
+    // This SHOULD be OK since we're filtering in loadPrescriptions
+    latestPrescription = filteredPrescriptions[0];
+    nrRefillsInProgress = filteredPrescriptions?.length;
+  }
+
+  const cardDetails = {
+    cardTitle: 'Prescription refills',
+    line1: latestPrescription?.attributes?.prescriptionName,
+    sectionTitle: 'Prescriptions',
+    line2: `Status: Submitted on ${moment(
+      latestPrescription?.attributes?.refillSubmitDate,
+    ).format('dddd, MMMM D, YYYY')}`,
+    line3: '',
+    ctaIcon: 'prescription-bottle',
+    ctaText: `${nrRefillsInProgress} prescription refill${
+      nrRefillsInProgress > 1 ? 's' : ''
```

Comment:
👍 In a past job I made a `pluralize` function that would have worked something like:
`pluralize('cactus', 1, 'cacti') => returns: 1 cactus` and `pluralize('cactus', 2, 'cacti') => returns: 2 cacti`. I think it also had the option to write out short numbers. I enjoy making helpers like that way too much 😄